libafl/src/monitors/tui/mod.rs

```diff
@@ -178,7 +218,46 @@ impl ClientTuiContext {
         self.corpus = client.corpus_size;
         self.objectives = client.objective_size;
         self.executions = client.executions;
-        self.exec_sec = exec_sec;
+        self.process_timing.client_start_time = client.start_time;
+        self.process_timing.last_new_entry = if client.last_corpus_time > client.start_time {
+            client.last_corpus_time - client.start_time
+        } else {
+            Duration::default()
+        };
+
+        self.process_timing.last_saved_solution = if client.last_objective_time > client.start_time
+        {
+            client.last_objective_time - client.start_time
+        } else {
+            Duration::default()
+        };
+
+        self.process_timing.exec_speed = exec_sec;
+
+        self.map_density = client
+            .get_user_stats("edges")
+            .map_or("0%".to_string(), ToString::to_string);
+
+        let default_json = serde_json::json!({
+            "pending": 0,
+            "pend_fav": 0,
+            "imported": 0,
+            "own_finds": 0,
+        });
+        let afl_stats = client
+            .get_user_stats("AflStas")
+            .map_or(default_json.to_string(), ToString::to_string);
+
+        let afl_stats_json: Value = serde_json::from_str(afl_stats.as_str()).unwrap();
```

Comment:
please try not to use unwrap() in lib
use return Result<(), Error> in this function